.github/workflows/tests.yml

```diff
@@ -87,8 +87,8 @@ jobs:
         # ignore php8.2 requirement error https://github.com/sebastianbergmann/phpunit/issues/5033
         composer-options: "--prefer-dist --no-progress --ignore-platform-req=php+"
 
-    - name: Setup Problem Matchers for PHPUnit
-      run: echo "::add-matcher::${{ runner.tool_cache }}/phpunit.json"
+    - name: Configure matchers
```

Comment:
setup-php problem matches does not support custom error messages
https://github.com/shivammathur/setup-php/discussions/451#discussioncomment-683241